Gems/Atom/RPI/Code/Source/RPI.Public/Pass/FullscreenTrianglePass.cpp

```diff
@@ -136,6 +136,12 @@ namespace AZ
             RHI::DrawLinear draw = RHI::DrawLinear();
             draw.m_vertexCount = 3;
 
+            if (m_shader == nullptr)
+            {
+                AZ_Error("PassSystem", false, "[FullscreenTrianglePass]: Shader not loaded!");
```

Comment:
Maybe put some more info to aid with debugging. I see in other places in that file they do:
`                AZ_Error("PassSystem", false, "[FullscreenTrianglePass '%s']: Failed to load shader '%s'!",
                    GetPathName().GetCStr(),
                    passData->m_shaderAsset.m_filePath.data());
`

Reply:
I'd have to add the block to get passData too. i.e.,:
```
if (m_shader == nullptr)
{
    // Load FullscreenTrianglePassData
    const FullscreenTrianglePassData* passData = PassUtils::GetPassData<FullscreenTrianglePassData>.(m_passDescriptor);
    if (passData == nullptr)
    {
        AZ_Error("PassSystem", false, "[FullscreenTrianglePass '%s']: Trying to construct without valid 
            FullscreenTrianglePassData!",
            GetPathName().GetCStr());
        return;
    }
    AZ_Error("PassSystem", false, "[FullscreenTrianglePass '%s']: Failed to load shader '%s'!",
        GetPathName().GetCStr(),
        passData->m_shaderAsset.m_filePath.data());
    return;
}
```
The LoadShader() would have already printed the same info earlier. I could still add GetPathName().GetCStr();

Reply:
Gotcha, thanks for checking.